package.json

```diff
@@ -66,6 +71,6 @@
     "vue-server-renderer": "^2.5.22"
   },
   "devDependencies": {
-    "husky": "1.3.1"
+    "husky": "0.14.3"
```

Comment:
We don't want to downgrade this, do we?

Reply:
Whoops, wasn't supposed to touch that-- that said we can probably upgrade to 2.2, the only 2.0 breaking change was the removal of node 6 support. See https://github.com/typicode/husky/blob/master/CHANGELOG.md.